docs/pages/setup/reference/resources.mdx

```diff
@@ -267,8 +267,8 @@ spec:
       'region': ['us-west-1', 'eu-central-1']
       # regular expressions start with ^ and end with $
       # Teleport uses golang regexp syntax.
```

Comment:
```suggestion
      # Teleport uses golang regexp syntax.
      # https://github.com/google/re2/wiki/Syntax
```

`Regular expressions start with ^ and end with $` is misleading, as there is no requirement that they do.

Additionally, we should link to the syntax.